ci/test/00_setup_env_native_asan.sh

```diff
@@ -17,10 +17,11 @@ else
 fi
 
 export CONTAINER_NAME=ci_native_asan
-export PACKAGES="systemtap-sdt-dev clang-18 llvm-18 libclang-rt-18-dev python3-zmq qtbase5-dev qttools5-dev-tools libevent-dev libboost-dev libdb5.3++-dev libminiupnpc-dev libnatpmp-dev libzmq3-dev libqrencode-dev libsqlite3-dev ${BPFCC_PACKAGE}"
+export PACKAGES="systemtap-sdt-dev clang-18 llvm-18 libclang-rt-18-dev python3-zmq qtbase5-dev qttools5-dev qttools5-dev-tools libevent-dev libboost-dev libdb5.3++-dev libminiupnpc-dev libnatpmp-dev libzmq3-dev libqrencode-dev libsqlite3-dev ${BPFCC_PACKAGE}"
```

Comment:
It follows the build docs: https://github.com/hebasto/bitcoin/blob/ecd23656db174adef61d3bd753d02698c3528192/doc/build-unix.md?plain=1#L84

Otherwise, CMakes fails to find `Qt5LinguistTools`.

Reply:
Right, please resolve.